geomstats/visualization.py

```diff
@@ -26,6 +27,21 @@
                'H2_poincare_disk', 'H2_poincare_half_plane', 'H2_klein_disk',
                'poincare_polydisk']
 
+EPS = 1e-8
+
+
+def nice_matplotlib():
```

Comment:
Nice is subjective, maybe use another name. Is this method necessary for the 2d ellipses only?